ix-dev/community/monerod/questions.yaml

```diff
@@ -563,7 +570,7 @@ questions:
                 description: CPUs limit for Monero Node.
                 schema:
                   type: int
-                  default: 2
```

Comment:
this will hard fail to deploy if the machine that is deployed does not have at least 4 cores.
docker does not allow exceeding